### ISSUE TITLE
Longer pause on paragraph breaks (Wavenet voices)

### ISSUE THREAD

**Issue by CyberShadow** [ken107/read-aloud#272]
Hi, thank you for this super cool extension.

I'm wondering if it would be possible to make the reader do a longer pause on a paragraph break. For example this is sometimes used as a form of emphasis, which is important for context.

If it matters, I'm using a WaveNet voice on Firefox.

**Comment by ken107** [comment]
This msut be done by the voice engine, which must intelligently know the context and how long to pause.  We can't do it progrmmatically, there's just no algorithm for it.

**Comment by CyberShadow** [comment]
Thank you for the response. Could there be an option which causes paragraphs to be submitted as separate chunks of text to the API? Then, the extension could insert a pause for as long as is needed.

**Comment by ken107** [comment]
Could you provide:
- The Wavenet voice you're using
- The URL of a webpage where you encounter very short paragraph pauses when using the above voice?

**Comment by CyberShadow** [comment]
Sure, here is an example that I can remember:

http://unsongbook.com/chapter-3-on-a-cloud-i-saw-a-child/ - the part starting with “WAIT, ARE YOU THAT SOHU?”

I'm using "ChatGPT English (fable)" and it is saying the “OH.” line quite quickly after the previous line, so it's not clear it's being said by another person so it becomes a little hard to follow when listening.

**Comment by CyberShadow** [comment]
Ah, sorry, I just saw that you asked about the WaveNet voice I used - unfortunately I don't remember what I had it set to at the time...

**Comment by ken107** [comment]
Thanks, we'll close this for now.  We're not getting any reports or requests about this, it's not a widespread issue.
